chef/cookbooks/nova_dashboard/templates/default/local_settings.py.erb

```diff
@@ -292,14 +292,14 @@ TIME_ZONE = "<%= @timezone %>"
 # Path to directory containing policy.json files
 #POLICY_FILES_PATH = os.path.join(ROOT_PATH, "conf")
```

Comment:
I wonder if we'd want to set this too, as it might be easier for people to put all the files in (say) /etc/horizon.